astropy/wcs/utils.py

```diff
@@ -14,13 +13,12 @@
            'proj_plane_pixel_area', 'is_proj_plane_distorted',
            'non_celestial_pixel_scales', 'skycoord_to_pixel',
            'pixel_to_skycoord', 'custom_wcs_to_frame_mappings',
-           'custom_frame_to_wcs_mappings', 'pixel_to_pixel']
+           'custom_frame_to_wcs_mappings', 'fit_wcs_from_points']
 
 
 def add_stokes_axis_to_wcs(wcs, add_before_ind):
     """
     Add a new Stokes axis that is uncorrelated with any other axes.
-
```

Comment:
Please revert these changes to the whitespace here and below - we like having the empty lines for readability :)